addons/sale_timesheet/models/sale_order.py

```diff
@@ -58,7 +58,7 @@ def action_view_timesheet(self):
             'search_default_billable_timesheet': True
         }  # erase default filters
         if self.timesheet_count > 0:
-            action['domain'] = [('so_line', 'in', self.order_line.ids)]
+            action['domain'] = [('so_line', 'in', self.order_line.ids), ('so_line.is_expense', '=', False)]
```

Comment:
```suggestion
            action['domain'] = [('so_line', 'in', self.order_line.ids), ('project_id', '!=', False)]
```
We could just check if the project is set to be sure to only select timesheets instead of any AAL with one of SOLs of the SO set. 🙂

Reply:
Oki, thanks, I've pushed your suggestion :)